ui-tests/src/test/resources/features/utils/debug-step.feature

```diff
@@ -0,0 +1,12 @@
+@debug-step
```

Comment:
Should this be included in PR also?

Reply:
Imenat so. perhaps it could be useful too also for others.

Reply:
Ok, maybe rename the file to `debug-step.feature.example` or `debug-step.example` something like that to be excluded from execution with whole testsuite.

Reply:
done